shm/src/Game.cpp

```diff
@@ -5,20 +5,21 @@
 #include <iostream>
 #include <memory>
 
-Game::Game(size_t money, size_t game_days, size_t final_goal)
+Game::Game(size_t money, size_t gameDays, size_t finalGoal)
     : money_(money)
-    , game_days_(game_days)
-    , final_goal_(final_goal)
+    , gameDays_(gameDays)
+    , finalGoal_(finalGoal)
     , time_(std::make_unique<Time>())
 {
-    current_day_ = time_->getElapsedTime();
+    size_t currentDay_ = time_->getElapsedTime();
```

Comment:
no, variable shadows private data member - let's delete the line as it is already initialised in Game.hpp (with zero which is the first day)